examples/tutorials/lines.py

```diff
@@ -1,5 +1,5 @@
 """
-Plot lines
+Plotting. lines
 ==========
```

Comment:
I forgot to mention that we should all keep the `=` same length as the title:
```suggestion
Plotting lines
==============
```